ui/src/components/Buttons/RunButton.tsx

```diff
@@ -58,10 +58,10 @@ export const RunButton: React.FC<Props> = ({ endpoint, payload, disabled }) => {
     if (!isBlob) {
       reqDetails.headers = { 'Content-Type': 'application/json' };
     }
-    const postUrlPrefix = new URL(
+    const url = new URL(
       `${window.location.protocol}//${window.location.hostname}:${window.location.port}`
     );
-    const url = new URL(`${postUrlPrefix}${postEndpoint}`);
```

Comment:
When stringifying `postUrlPrefix` (which was a `URL`), it would add an extra slash on the end. Because `postEndpoint` also begins with a slash, this would end up with two slashes.